locust_cloud/utils.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import site
+from modulefinder import ModuleFinder
+from pathlib import Path
+
+logger = logging.getLogger(__name__)
+
+CWD = Path.cwd()
+IMPORTS_EXCLUDE_PATHS = [Path(p) for p in [site.getusersitepackages(), *site.getsitepackages()]]
+
+
+def get_imported_files(file_path: Path) -> set[Path]:
+    imports: set[Path] = set()
+
+    script_path = Path(file_path).resolve()
+
+    mf = ModuleFinder()
+    try:
+        mf.run_script(str(script_path))
+    except Exception as e:
+        logger.exception(e)
+        logger.error(f"{script_path} is not a valid Python script")
+        return set()
+
+    for mod in mf.modules.values():
+        path = getattr(mod, "__file__", None)
+        if not path:
+            continue  # built-ins, etc.
+
+        p = Path(path).resolve()
+        if (
```

Comment:
I prefer this if statement inverted: list the conditions to include instead of excluding the file. Also results in one less continue :)